estuary/utils/story.py

```diff
@@ -185,6 +188,13 @@ def get_wait_times(self, results):
 
             next_artifact_start_time = datetime.strptime(next_artifact.timeline_timestamp,
```

Comment:
Optional: If you put this in the else of the if statement below, you avoid needing to parse the timestamp.